packages/lit-dev-content/site/tutorials/content/reactivity/00.md

```diff
@@ -0,0 +1,14 @@
+Reactive properties and the reactive update cycle are an important part of the Lit library. Updating a reactive property can trigger a reactive update cycle, which in turn produces side effects—like re-rendering the element, updating its attributes, or firing events.
+
+If you did the [Intro to Lit](/tutorials/intro-to-lit/) tutorial, you've already used some reactive properties. This tutorial digs a little deeper into reactive properties and the reactive update cycle:
+
+*   How reactive properties trigger an update (and how to triggering an update manually).
+*   Customizing when the component updates.
```

Comment:
I think performing an action, a custom side-effect, when a property changes is more important to explain than `shouldUpdate` or `hasChanged`.

This would be implemented via `updated`, and could be just as basic as alert|console.log(`${this.name}!`)

Reply:
I'd be happy to add an `updated()` example. It felt kind of strange not having something there. But I had a hard time finding reasonable use cases.

A console log or alert would demonstrate that `updated()` exists, but not why you'd use it.

The only example we've mentioned so far is measuring the element. (You also mention that changing a property in `updated` triggers a new update, so presumably there's a place that's useful, like triggering a transition animation?)

Reply:
OK, added an `updated()` example that does a simple animation in a not-very simple way.